src/http.ts

```diff
@@ -1660,16 +1824,19 @@ export class LemmyHttp {
     type_: HttpType,
     endpoint: string,
     form: BodyType,
+    options: RequestOptions | undefined,
   ): Promise<ResponseType> {
     let response: Response;
     if (type_ === HttpType.Get) {
       const getUrl = `${this.#buildFullUrl(endpoint)}?${encodeGetParams(form)}`;
       response = await this.#fetchFunction(getUrl, {
+        ...options,
```

Comment:
When `options` is `undefined`, woudln't spreading it cause an error?

Reply:
No, it's ok :) I think it is a problem for spreading on arrays, not objects. https://stackoverflow.com/a/62885880/1319878

> However, {...x} requires x to be enumerable because it needs not just values, but keys along with them. An object is a prime example of an enumerable.
> undefined is enumerable. undefined is sometime treated as an object, and this is one of those cases. Objects are enumerable because they can have properties. {...undefined} therefore succeeds. for (const p in undefined) similarly succeeds, since in requires an enumerable.